src/mmfatl.c

```diff
@@ -34,7 +125,8 @@ void test_mmfatl(bool* ok)
     bool silent = TEST_SILENT;
 
     *ok = *ok
+        && test_initBuffer(silent)
```

Comment:
Can you have each successful test log that it was successful in CI? We don't need the details of the test but it's important to know that the test itself ran (compare with the regular test suite).

Reply:
This will in the end be awfully verbose, because my initial PR #86 already had 30+ individual tests.  I don't recommend this.  But one message for each module is ok, I think

Reply:
In case of suspicion you can still checkout the sources, uncomment the # define TEST_FORCE_ENABLE_LINE and compile it the usual way with ./build.sh -c .  This will run the regression test with all the gory  details.

Reply:
> This will in the end be awfully verbose, because my initial PR #86 already had 30+ individual tests. I don't recommend this. But one message for each module is ok, I think

Yes, I expect to see 30 lines in CI. This is pretty standard for testing frameworks. 30 lines is not a lot, especially if you use some colors to make things stand out, or use very similar formatting for each test line so that a failing test looks very different from the surrounding code.

Reply:
30 lines for each module (i.e mmxxxx.c file) sums up to several 100

Reply:
I think I'll make it configurable.